### PR TITLE
Fixed "Build your own tool" page

### LINKED ISSUE #586: Update "Build your own tool" docu
- It references an old extension pack
- It excludes the built in git extension

### PR DESCRIPTION
**Pull request by JonasHelming**
fixed #586

### CODE REVIEW

**Comment by JonasHelming** [comment]
@tsmaeder This should fix the immediate issue. If you are fine with the changes, feel free to directly merge and link this on the two issues you mentioned.
Mid-term, we should consider indeed to get rid of the manual example and fully support Yeoman, I will add it to the agenda of the dev call again.

**Comment by JonasHelming** [comment]
@tsmaeder Ping

File: src/docs/composing_applications.md [hunk 1]
```diff
@@ -119,7 +117,7 @@ An example `package.json` may look like the following:
 The following properties are used to consume built-in plugins (bundled extensions):
 
 - `theiaPluginsDir`: the relative path to deploy plugins into
-- `theiaPlugins`: the collection of plugins to download (individual plugins or extension-packs) - can point to any valid download URL (ex: Open VSX, GitHub Releases, etc.)
+- `theiaPlugins`: the collection of plugins to download (individual plugins or extension-packs) - can point to any valid download URL (ex: Open VSX, GitHub Releases, etc.). **In the example above, replace ${downloadURL} with a link to the latest builtins pack that can be found [on openVSX](https://open-vsx.org/extension/eclipse-theia/builtin-extension-pack) (copy the link on the download button). See also [here](https://github.com/eclipse-theia/theia/blob/a9345dab1b6ea175ec98a5bf197e56876f2d58f6/package.json#L105) for an example**
```
Comment (tsmaeder) [inline src/docs/composing_applications.md:120]: Don't we want to point at `master` here? https://github.com/eclipse-theia/theia/blob/master/package.json

Comment (JonasHelming) [inline src/docs/composing_applications.md:120]:
```suggestion
- `theiaPlugins`: the collection of plugins to download (individual plugins or extension-packs) - can point to any valid download URL (ex: Open VSX, GitHub Releases, etc.). **In the example above, replace ${downloadURL} with a link to the latest builtins pack that can be found [on openVSX](https://open-vsx.org/extension/eclipse-theia/builtin-extension-pack) (copy the link on the download button). See also the respective section [in this file](https://github.com/eclipse-theia/theia/blob/master/package.json) for an example**
```

Comment (JonasHelming) [reply src/docs/composing_applications.md]: fixed

**Review by tsmaeder — APPROVED** [review]

**Comment by JonasHelming** [comment]
@tsmaeder please link this on the two issues you mentioned.

**Pull request merged**